games/snes.rec

```diff
@@ -0,0 +1,60 @@
+%rec: snes
+
+title: Super Mario World
+year: 1990
+
+title: F-Zero
+year: 1990
+
+title: The Legend of Zelda: A Link to the Past
+year: 1991
+
+title: Super Castlevania IV
+year: 1991
+
+title: Final Fantasy IV 
```

Comment:
3 FF's seems like a lot, which one is the best?

Reply:
VI is the best!

Between V and IV it's tricky. V has better gameplay and IV has better story. I'll mull it over.